chains/ethereum/methods.go

```diff
@@ -0,0 +1,105 @@
+package ethereum
+
+import (
+	"math/big"
+
+	msg "github.com/ChainSafe/ChainBridgeV2/message"
+	"github.com/ChainSafe/log15"
+)
+
+const StoreMethod = "store"
+const CreateDepositProposalMethod = "createDepositProposal"
+const VoteDepositProposalMethod = "voteDepositProposal"
+const ExecuteDepositMethod = "executeDeposit"
+
+// TODO: make this configurable
+var gasLimit = big.NewInt(6721975)
+var gasPrice = big.NewInt(20000000000)
```

Comment:
In the ticket that is making this configurable (#136 ), it pulls from the Opts from the chainconfig. Shouldn't be too much of an issue, but will cause a need for a refactor when both issues are merged.